preprocess/code/variable_display_util.py

```diff
@@ -31,202 +46,116 @@ def get_default_settings():
-              "created":"..time stamp..",
-              "preprocess_id":45,
-              "data_url":"http://metadata.2ravens-url.org/preprocess/data/45",
-              "format":"jsonschema",
-              "preprocess_version":"1-0-0",
-              "schema_version":"1.0.0"
-           },
-           "variables":{
-              "cylinders":{
-                 "label":"",
-                 "mean":213,
-                 "median":34
-              }
-           },
-           "variable_display":{
-              "editable":[
-                 "label",
-                 "varSumState"
-              ],
-              "cylinders":{
-                 "viewable":"true",
-                 "omit":[
-                    "numChar",
-                    "mean"
-                 ],
-                 "images":{
-                    "url1":{
-                       "type":"pdf",
-                       "description":"This is the density of the mpg variable.",
-                       "dateCreated":"timestamp",
-                       "version":"imagecreate v0.1"
-                    }
-                 }
-              }
-           }
-        }
-        :return:
-        """
-        original_json = \
-            {
-               "$schema":"http://(link to eventual schema)/jjonschema/1-0-0#",
-               "self":{
-                  "description":"TwoRavens metadata generated by ....",
-                  "created":"..time stamp..",
-                  "preprocess_id":45,
-                  "data_url":"http://metadata.2ravens-url.org/preprocess/data/45",
-                  "format":"jsonschema",
-                  "preprocess_version":"1-0-0",
-                  "schema_version":"1.0.0"
-               },
-               "variables": {
-                  "cylinders": {
-                     "numchar":"continuous",
-                      "nature": "nominal",
-                     "mean":213,
-                     "median":34
-
-                  },
-                  "mpg":{
-                      "numchar": "continuous",
-                       "nature": " ordinal",
-                       "mean": 313,
-                       "median": 54
-
-                   }
-               },
-               "variable_display":{
-                  "cylinders":{
-                     "viewable":"true",
-                     "omit":[
-                        "numChar",
-                        "mean"
-                     ],
-                     "label":"",
-                     "images":{
-                        "url1":{
-                           "type":"pdf",
-                           "description":"This is the density of the mpg variable.",
-                           "dateCreated":"timestamp",
-                           "version":"imagecreate v0.1"
-                        }
-                     }
-                  }
-               }
-            }
-
-        return json.dumps(original_json, indent=4, cls=NumpyJSONEncoder)
-
     def var_display(self):
```

Comment:
Change name of function.  e.g. `update_preprocess_data`

Reply:
For the overall function, return a tuple instead of a single value.
- if it fails:
   `return False, self.error_messages`
- if it succeeds:
   `return True, self.final_original_output()`

Usage for code calling this would be:
 ```python
success, updated_json_or_err = VariableDisplayUtil(orig_metadata, update_json)
if success:
     # return appropriate message; updated data is in "updated_json_or_err"
else:
    # return error message; [] of error messages is in updated_json_or_err
```
This will change some other code, but make it easier to use